autoload/neomake/makers/ft/svelte.vim

```diff
@@ -0,0 +1,28 @@
+" vim: ts=4 sw=4 et
+function! neomake#makers#ft#svelte#EnabledMakers() abort
+    let makers = []
+    if executable('svelte-check')
+        call add(makers, 'svelte_check')
+    endif
+    if executable('eslint')
+        call add(makers, 'eslint')
+    endif
```

Comment:
1. Note that the executable check (based on `maker.exe`) gets done automatically, so this could be removed here.

2. Note also that it might not be wanted that makers get enabled/used by default (https://github.com/neomake/neomake/issues/2515).  I think it makes sense to have them here enabled by default though.